frontend/src/app/components/filters/query-filters/query-filters.component.ts

```diff
@@ -122,7 +118,7 @@ export class QueryFiltersComponent extends UntilDestroyedMixin implements OnInit
 
   public get isSecondSpacerVisible():boolean {
     const hasSearch = !!_.find(this.filters, (f) => f.id === 'search');
-    const hasAvailableFilter = !!_.find(this.filters, (f) => f.id !== 'search' && this.isFilterAvailable(f.id));
+    const hasAvailableFilter = !!_.find(this.filters, (f) => f.id !== 'search' && this.isFilterAvailable(f));
```

Comment:
No need for lodash here, since it is an array you can use the array.find native method.